ansible/roles/stack-sunbird/templates/inbound.env

```diff
@@ -49,5 +49,5 @@ ENV={{env}}
 #Redis
 REDIS_HOST={{sunbird_redis_host}}
 REDIS_PASS=""
```

Comment:
Please add a variable as a placeholder like this - https://github.com/project-sunbird/sunbird-devops/blob/master/ansible/roles/stack-sunbird/templates/sunbird_nodebb.env#L11

Reply:
is there a variable available for redis password like "sunbird_redis_pass", or should I just add a new one here.
Something like this should work right?
REDIS_PASS={{sunbird_redis_pass | default('')}}